modules/ocf/manifests/firewall/firewall46.pp

```diff
@@ -0,0 +1,18 @@
+define ocf::firewall::firewall46($opts,) {
```

Comment:
Why not just add an option here to set `$before` to `undef` by default so that you don't have to keep specifying it everywhere in the same way? Same with the output chain, seems like it would be easier to have a separate class for that so that there's less repetition in `$opts`.

Reply:
We wouldn't want to set `$before` to `undef` by default, because we purposely set the default otherwise in `site.pp`:

    Firewall {
      before => Class['ocf::firewall::post'],
    }

As for your other comment, it could be looked into, although I'm not sure it adds much value over writing another `chain => 'PUPPET-OUTPUT'`.